File: backend/src/main/java/es/dawgrupo2/zendashop/extendedDTO/OrderItemExtendedDTO.java
package es.dawgrupo2.zendashop.extendedDTO;

import es.dawgrupo2.zendashop.basicDTO.GarmentBasicDTO;
import es.dawgrupo2.zendashop.basicDTO.OrderBasicDTO;

public record OrderItemExtendedDTO (
    Long id,
    GarmentBasicDTO garment,
    int quantity,
    String size,
    OrderBasicDTO order
){
    
}

File: frontend/app/dtos/OrderItemExtendedDTO.ts
import type GarmentBasicDTO from "./GarmentBasicDTO";
import type OrderBasicDTO from "./OrderBasicDTO";

export default interface OrderItemExtendedDTO {
  id: number;
  garment: GarmentBasicDTO;
  quantity: number;
  size: string;
  order: OrderBasicDTO;
}
